src/scantailor/core/settings.py

```
@@ -0,0 +1,229 @@
+    @classmethod
+    def load(cls) -> ApplicationSettings:
+        """Load settings from the config file.
+
+        Returns:
+            ApplicationSettings loaded from file, or defaults if file doesn't exist.
+        """
+        config_path = cls.get_config_path()
+        if config_path.exists():
+            try:
+                return cls.model_validate_json(config_path.read_text(encoding="utf-8"))
+            except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)